common/lib/xmodule/xmodule/js/src/video/05_video_quality_control.js

```diff
@@ -4,27 +4,27 @@
 'use strict';
 define(
 'video/05_video_quality_control.js',
-[],
-function () {
-    var template = [
-        '<button class="control quality-control is-hidden" aria-disabled="false" title="',
-            gettext('High Definition'),
-        '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
-                // Translator note:
-                // HD stands for high definition
+['edx-ui-toolkit/js/utils/html-utils'],
+function (HtmlUtils) {
+    var template = HtmlUtils.interpolateHtml(
+        HtmlUtils.HTML([
+            '<button class="control quality-control is-hidden" aria-disabled="false" title="',
+                '{highDefinition}',
+            '">',
+                '<span class="icon icon-hd" aria-hidden="true">HD</span>',
                 '<span class="sr text-translation">',
-                    gettext('High Definition'),
+                    '{highDefinition}',
                 '</span>&nbsp;',
-                '<span class="text control-text">',
-                    // Translator note:
-                    // Values are 'off' or 'on' depending on the state of the HD control
-                    gettext('off'),
+                '<span class="sr control-text">',
```

Comment:
Keep this `sr` though because it provides additional context for non-visual learners, who may not be able to see that the HD is blue when enabled.